File: examples/python/testing-patterns/basic-tests/verify-simple-values.py
from test.conftest import deserialize_operation_payload

@pytest.mark.durable_execution(
    handler=calculator_handler,
    lambda_function_name="calculator",
)
def test_calculation_result(durable_runner):
    """Test calculation returns correct result."""
    with durable_runner:
        result = durable_runner.run(input={"a": 5, "b": 3}, timeout=10)
    
    assert result.status is InvocationStatus.SUCCEEDED
    assert deserialize_operation_payload(result.result) == 8

File: examples/java/testing-patterns/basic-tests/verify-simple-values.java
@Test
void testCalculationResult() {
    var runner = LocalDurableTestRunner.create(Map.class, new CalculatorHandler());

    var result = runner.runUntilComplete(Map.of("a", 5, "b", 3));

    assertEquals(ExecutionStatus.SUCCEEDED, result.getStatus());
    assertEquals(8, result.getResult(Integer.class));
}
